WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -36,6 +36,9 @@ public PrivacySuggestionsFlyoutViewModel()
 
 	public async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo info, BitcoinAddress destination, BuildTransactionResult transaction, bool isFixedAmount, CancellationToken cancellationToken)
 	{
+		using CancellationTokenSource childCTS = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken);
+		childCTS.CancelAfter(15000);
```

Comment:
I think it's better to do it this way:

```suggestion
		using CancellationTokenSource timeoutCts = new(TimeSpan.FromSeconds(15));
		using CancellationTokenSource linkedCts = CancellationTokenSource.CreateLinkedTokenSource(timeoutCts.Token, cancellationToken);
```